Dockerfile.test.cpu

```diff
@@ -74,21 +75,21 @@ RUN if [[ ${MPI_KIND} == "OpenMPI" ]]; then \
             cd /usr/local && tar -zxf /tmp/openmpi-3.0.0-bin.tar.gz && ldconfig && \
             echo "mpirun -allow-run-as-root -np 2 -H localhost:2 -bind-to none -map-by slot -mca mpi_abort_print_stack 1" > /mpirun_command; \
     elif [[ ${MPI_KIND} == "ONECCL" ]]; then \
-        wget --progress=dot:mega -O /tmp/oneccl.tar.gz https://github.com/oneapi-src/oneCCL/archive/master.tar.gz && \
+        wget --progress=dot:mega -O /tmp/oneccl.tar.gz https://github.com/shirosankaku/oneCCL/archive/${CCL_PACKAGE}.tar.gz && \
```

Comment:
Is this the right org for this repo?  Should it be `oneapi-src`?

Reply:
@tgaddair - we will replace fork URL by official repo URL right after oneCCL Gold release in few weeks
we will notify you
before that please review PR and keep it open